caplena/endpoints/projects_endpoint.py

```diff
@@ -152,6 +155,38 @@ def append_rows(
 
         return self.build_response(response, resource=RowsAppend)
 
+    @ttl_cache(ttl=TTL_STATUS_CACHE_EXPIRE)
+    def get_append_status(
+        self, *, project_id: str, task_id: Optional[Union[uuid.UUID, str]] = None
+    ) -> "RowsAppendStatus":
+        """Checks statuses of all upload tasks for requested project
+        or only requested upload task if it's ID is provided
+
+        Note: We keep the status of upload tasks only for the 7 days. After that time task info won't be available.
+
+        :param project_id: The project identifier.
+        :param task_id: Task id which status will be checked
```

Comment:
Maybe mention here as well what's already mentioned in the description? task_id is optional, if it's missing all upload task for the current project will be checked.